### DISCUSSION TITLE
How to remove the default theme applied by Chakra UI?

### DISCUSSION THREAD

**Discussion by FC5570** [chakra-ui/chakra-ui#4926] [Q&A] [answered]
Hello. I hope this is the right place to ask this type of question.

I'm using @chakra-ui/react with Tailwind CSS and NextJS. I have set my background color to `black` in my `globals.css` file:
```css
body {
    background-color: black;
}
```

But I don't see the black color being applied, I only see a white screen with the content. 

This is my app.js file:
```js
import { ChakraProvider } from '@chakra-ui/react'
import 'tailwindcss/tailwind.css'
import '../styles/globals.css' // file which sets the body's background-color to black

function MyApp({ Component, pageProps }) {
  return (
    <ChakraProvider>
      <Component {...pageProps} />
    </ChakraProvider>
  )
}

export default MyApp
```

I assume this is because of chakra's default theme? How would I disable it?

Thank you!

**Comment by no-yan** [comment, marked as answer]
The reason is that chakra UI has body-bg style to global in the [default theme](https://github.com/chakra-ui/chakra-ui/blob/78d9c30e6b9477080c75b2e601394a21ed93fcf2/packages/theme/src/styles.ts#L8). You can override this with the following code
https://codesandbox.io/s/zeta-heading-size-issue-forked-w4xn8?file=/src/index.js
```
const theme = extendTheme({
  styles: {
    global: () => ({
      body: {
        bg: "",
      },
    }),
  },
});
```

**Reply by FC5570** [reply]
Thank you!

**Reply by Djougabench** [reply]
it doesnt work

**Reply by alaska102** [reply]
Did you ever figure it out?

**Reply by Suvraneel** [reply]
Perfect ... GG 🔥 :octocat:

**Reply by schkolne** [reply]
worked for me when i wrote "background" instead of "bg"

**Reply by tombohub** [reply]
is there a script or something to remove all of the styling?

**Reply by JaredSystems** [reply]
Worked for me! Thanks, been tackling this for hours

**Reply by 0su0000000i17** [reply]
Thank you!

**Comment by TanBeige** [comment]
How can you remove all of the default theme though? I have a project that already uses its own CSS and we are starting to integrate Chakra but it is messing up some layouts. If there's a way to import specific components from Chakra as well that would be beneficial.

**Reply by PabloVMartins** [reply]
I passed the resetCSS={false} prop in chakraProvider, and it solved my problem with CSS.

**Comment by OxBryte** [comment]
i need help pls
been having issues with chakra lately 
first is what i'm working on needs me to downgrade react to aparticular version so i have to downgrade chakra also 
and i'm using chakra v1.1.0 at the moment 
so whenever i host the project on vercel i get this error and i have tried everything still same 
here is an image
![image](https://github.com/chakra-ui/chakra-ui/assets/62702022/921b5c52-98e2-4a96-9201-535b78b31da6)

**Reply by JayJ-WeCode** [reply]
Have you found a solution to this problem?

**Comment by OSAMA263** [comment]
this is if u want to diable the theme

```
delete theme.styles.global;
export default App() {
  return (
   <ChakraProvider theme={theme} resetCSS={false} disableGlobalStyle={true}>
      {* your content *}
   </ChakraProvider>
  );
}
```

**Comment by sbbeez** [comment]
I wasnt using any theming capabilities of `chakra-ui`,

so just setting `disableGlobalStyle` in `<ChakraProvider>{children}</ChakraProvider>` help fix the issue.
